package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "corsproxy",
+  "version": "0.0.2",
```

Comment:
just wondering, why did you add the version here, and why 0.0.2? Note that version is set dynamically by [semantic-release](https://github.com/semantic-release/semantic-release)